### ISSUE TITLE
--publish error message is not clear

### ISSUE THREAD

**Issue by benjamin-bergia** [containers/libpod#4804]
**Description**
When trying to publish on a port that is already used by an other container, the following error is returned:
`Error: error from slirp4netns while setting up port redirection: map[desc:bad request: add_hostfwd: slirp_add_hostfwd failed]`. It would be very helpful to have a specific error message for this type of error.

**Steps to reproduce the issue:**

1.  podman run -it --rm -p 8080:80 wordpress

2. podman run -it --rm -p 8080:80 wordpress

**Describe the results you received:**
Error: error from slirp4netns while setting up port redirection: map[desc:bad request: add_hostfwd: slirp_add_hostfwd failed]

**Describe the results you expected:**
Something in the line of "Error: port 8080 already in use

**Output of `podman version`:**
```
Version:            1.6.2
RemoteAPI Version:  1
Go Version:         go1.13.1
OS/Arch:            linux/amd64
```

**Output of `podman info --debug`:**

```
debug:
  compiler: gc
  git commit: ""
  go version: go1.13.1
  podman version: 1.6.2
host:
  BuildahVersion: 1.11.3
  CgroupVersion: v2
  Conmon:
    package: conmon-2.0.2-1.fc31.x86_64
    path: /usr/bin/conmon
    version: 'conmon version 2.0.2, commit: 186a550ba0866ce799d74006dab97969a2107979'
  Distribution:
    distribution: fedora
    version: "31"
  IDMappings:
    gidmap:
    - container_id: 0
      host_id: 1000
      size: 1
    - container_id: 1
      host_id: 100000
      size: 65536
    uidmap:
    - container_id: 0
      host_id: 1000
      size: 1
    - container_id: 1
      host_id: 100000
      size: 65536
  MemFree: 358207488
  MemTotal: 8083656704
  OCIRuntime:
    name: crun
    package: crun-0.10.6-1.fc31.x86_64
    path: /usr/bin/crun
    version: |-
      crun version 0.10.6
      spec: 1.0.0
      +SYSTEMD +SELINUX +APPARMOR +CAP +SECCOMP +EBPF +YAJL
  SwapFree: 7967862784
  SwapTotal: 8233414656
  arch: amd64
  cpus: 4
  eventlogger: journald
  hostname: pcrcsisbb
  kernel: 5.3.12-300.fc31.x86_64
  os: linux
  rootless: true
  slirp4netns:
    Executable: /usr/bin/slirp4netns
    Package: slirp4netns-0.4.0-20.1.dev.gitbbd6f25.fc31.x86_64
    Version: |-
      slirp4netns version 0.4.0-beta.3+dev
      commit: bbd6f25c70d5db2a1cd3bfb0416a8db99a75ed7e
  uptime: 1h 44m 24.57s (Approximately 0.04 days)
registries:
  blocked: null
  insecure: null
  search:
  - docker.io
  - registry.fedoraproject.org
  - registry.access.redhat.com
  - registry.centos.org
  - quay.io
store:
  ConfigFile: /home/b/.config/containers/storage.conf
  ContainerStore:
    number: 1
  GraphDriverName: overlay
  GraphOptions:
    overlay.mount_program:
      Executable: /usr/bin/fuse-overlayfs
      Package: fuse-overlayfs-0.7-1.fc31.x86_64
      Version: |-
        fusermount3 version: 3.6.2
        fuse-overlayfs: version 0.7
        FUSE library version 3.6.2
        using FUSE kernel interface version 7.29
  GraphRoot: /home/b/.local/share/containers/storage
  GraphStatus:
    Backing Filesystem: extfs
    Native Overlay Diff: "false"
    Supports d_type: "true"
    Using metacopy: "false"
  ImageStore:
    number: 68
  RunRoot: /run/user/1000
  VolumePath: /home/b/.local/share/containers/storage/volumes
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
podman-1.6.2-2.fc31.x86_64
```
 /kind bug

**Comment by vrothberg** [comment]
@giuseppe @AkihiroSuda PTAL

**Comment by mheon** [comment]
I think this one might be fixed on 1.7.0

**Comment by AkihiroSuda** [comment]
Being fixed in https://github.com/containers/libpod/pull/4592. 

Now the error message is like `failed to expose ports via rootlessport: "listen tcp 0.0.0.0:8080: bind: address already in use`
